Lib/ast.py

```diff
@@ -62,11 +62,13 @@ def literal_eval(node_or_string):
         node_or_string = parse(node_or_string, mode='eval')
     if isinstance(node_or_string, Expression):
         node_or_string = node_or_string.body
+    def _malformed(node):
```

Comment:
Please, call this function with a verb, not a noun. Finding `malformed` in the middle of the code is a bit surprising compared with `report_malformed_node` or `raise_malformed_exception` or something similar.